pkg/api/annotation_key_constants.go

```diff
@@ -72,6 +72,10 @@ const (
 	// This annotation can be attached to node.
 	ObjectTTLAnnotationKey string = "node.alpha.kubernetes.io/ttl"
 
+	// CheckpointAnnotationKey represents a Resource (Pod) that should be checkpointed by
+	// the kubelet prior to running
+	CheckpointAnnotationKey string = "node.kubernetes.io/checkpoint"
```

Comment:
Nope alpha has been removed from the naming conventions per api-machinery folks.